backend/entities/Pims.Dal.Entities.csproj

```diff
@@ -8,6 +8,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.5" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
We shouldn't need Newtonsoft, as we don't use it anywhere else for serialization.  Use the built in System.Text.Json.

Reply:
[JsonObject(MemberSerialization.OptIn)] is only part of Newtonsoft, let me see if there is an equivalent in System.Text.Json.Serialization

Reply:
I doubt our serialization would obey a Newtonsoft specific attribute.